src/unlimited.cpp

```diff
@@ -210,12 +209,12 @@ void HandleExpeditedRequest(CDataStream& vRecv,CNode* pfrom)
             if (xpeditedBlk.size() < maxExpedited )
 	      {
 		LogPrint("blk", "Starting expedited blocks to peer %s (%d).\n", pfrom->addrName.c_str(),pfrom->id);
-		std::vector<CNode*>::iterator it = std::find(xpeditedBlk.begin(), xpeditedBlk.end(),((CNode*)NULL));
+		std::vector<CNode*>::iterator it = std::find(xpeditedBlk.begin(), xpeditedBlk.end(),(CNode*)NULL);  // find an empty array location
```

Comment:
Comment in its own line. This is the very typical example as to why. The code appears changed when it isn't.